sdcm/cluster.py

```diff
@@ -4996,16 +4996,18 @@ def install_scylla_monitoring_prereqs(node):  # pylint: disable=invalid-name
                 sudo apt-get update
                 sudo apt-get install -y docker docker.io
                 apt-get install -y software-properties-common
-                add-apt-repository -y ppa:deadsnakes/ppa
-                apt-get update
-                apt-get install -y python3.6 python3.6-dev
+                apt-get install -y python3 python3-dev
                 apt-get install -y python-setuptools unzip wget
-                apt install -y python3-pip
-                python3.6 -m pip install --upgrade pip
-                python3.6 -m pip install pyyaml
-                pip3 install -I -U psutil
+                apt-get install -y python3-pip
+                python3 -m pip install pyyaml
+                python3 -m pip install -I -U psutil
                 systemctl start docker
             """)
+            if node.distro.is_ubuntu18:
+                prereqs_script += dedent("""
+                    add-apt-repository -y ppa:deadsnakes/ppa
```

Comment:
now you are not using the version out of deadsnake at all, 
i.e. this part can be remove and everything would work

the whole idea of use deadsnakes repo, was to install python 3.6, which wasn't available at some point, if it's available and everything works on ubuntu 18, no point of adding that repo at all.

Reply:
Then I guess it's just unnecessary completely. I'll remove it.